org.wso2.carbon.identity.organization.mgt.core/src/main/java/org/wso2/carbon/identity/organization/mgt/core/dao/CacheBackedOrganizationMgtDAO.java

```diff
@@ -236,6 +272,8 @@ private void clearOrganizationCache(int tenantId, String organizationId) throws
             }
             OrganizationCacheKey cacheKey = new OrganizationCacheKey(organizationId);
             organizationCache.clearCacheEntry(cacheKey);
+            organizationCache.clearCacheEntry(
+                    new OrganizationCacheKey(String.format(ALL_LEVEL_CHILD_ORG_LIST_CACHE_KEY_FORMAT, organizationId)));
```

Comment:
Will this cache get invalidated/updated upon org add/deletion?

Reply:
yes